download/index.md

```diff
@@ -66,8 +146,8 @@ A pre-built package that contains Apache Brooklyn and all of its dependencies in
 </div>
 
 <div class="text-center">
-  <a class="btn btn-default" href="https://www.apache.org/dyn/closer.lua?action=download&filename=brooklyn/apache-brooklyn-{{ site.brooklyn-stable-version }}/apache-brooklyn-{{ site.brooklyn-stable-version }}-bin.tar.gz" role="button">Tarball</a>
-  <a class="btn btn-default" href="https://www.apache.org/dyn/closer.lua?action=download&filename=brooklyn/apache-brooklyn-{{ site.brooklyn-stable-version }}/apache-brooklyn-{{ site.brooklyn-stable-version }}-bin.zip" role="button">Zip</a>
+  <a class="btn btn-default" href="https://www.apache.org/dyn/closer.lua?action=download&filename=brooklyn/apache-brooklyn-1.0.0-M1/apache-brooklyn-1.0.0-M1-bin.tar.gz" role="button">Tarball</a>
```

Comment:
Shouldn't we use `{{site.brooklyn-stable-version}}` there? Same question for the remaining versions below

Reply:
The way this is structured is to have two sets of files, one at the top for 0.12.0 which is still the official recommended version, and then an equivalent set below for 1.0.0-M1. This is to match up with what is currently on https://brooklyn.apache.org/download/index.html. This line is just the start of the links for 1.0.0-M1.

Reply:
well actually they start a bit above

Reply:
Ahaa, I didn't realised there was 2 columns as I only scanned the code. Looks good then!